services/fis/src/fis/adapters/inbound/fastapi_/routes.py

```diff
@@ -120,9 +149,12 @@ async def ingest_secret(
     _token: Annotated[IngestTokenAuthContext, require_token],
 ):
     """Decrypt payload and deposit file secret in exchange for a secret id"""
-    file_secret = await upload_metadata_processor.decrypt_secret(
-        encrypted=encrypted_payload
-    )
+    try:
+        file_secret = await upload_metadata_processor.decrypt_secret(
+            encrypted=encrypted_payload
+        )
+    except DecryptionError as error:
+        raise HTTPException(status_code=422, detail=str(error)) from error
```

Comment:
Should be documented in the FastAPI header

Reply:
422 should be documented by default, but once I find where to get the HTTPValidationError model from, I can adjust the description.

Reply:
Added in cf8cdd1a079e0b857e9437ff57bd175790144104